packages/react/tsconfig.json

```diff
@@ -26,9 +26,6 @@
     "importHelpers": true,
     "composite": true,
     "baseUrl": "./",
-    "rootDir": "./src",
-    "paths": {
-      "@clr/core/*": ["../../dist/core/*"] // This mapping is relative to "baseUrl"
-    }
```

Comment:
this `paths` is not needed anymore since we're copying over the `@clr/core` to this package's `node_modules`.

Reply:
It looks like the tests are failing because it cant find the package. I think its because of this in the `jest.config.js`
I dont think thats needed anymore and will fix the tests.

```
  moduleNameMapper: {
    '@clr/(.*)': '<rootDir>/../core/dist/$1',
  },
```

Also it looks like there is a similar path mapping in the `tsconfig.build.json` file that probably be removed as well.

Reply:
Just a note: I removed them from both `tsconfig.json` and `tsconfig.build.json` but kept the one in jest config. Removing it from jest config caused the build to fail even though test runs fine locally. Local env might be relying on a pre-condition that build isn't replicating.